src/index.ts

```diff
@@ -757,7 +757,11 @@ const getAllInputFilesUnderReadme = (readMePath: string): asyncIt.AsyncIterableE
     const dir = path.dirname(readMePath)
     yield* tscommonFs
       .recursiveReaddir(dir)
-      .filter(filePath => path.extname(filePath) === '.json')
+      .filter(
+        filePath =>
+          path.extname(filePath) === '.json' &&
+          filePath.split(path.sep).some(folder => folder === 'preview' || folder === 'stable'),
```

Comment:
This looks like it will find any folder with that name so if we had something like `example\abc\preview\e1.json` it would include. Should we try to scope it to the top level folder?

Reply:
I might have a better fix that doesn't require changing avocado code, just config:

https://github.com/Azure/azure-rest-api-specs/pull/34878/files